Core/EventHandlers.lua

```diff
@@ -2071,6 +2077,25 @@ end
 
 local IsQuestFlaggedCompleted = C_QuestLog.IsQuestFlaggedCompleted
 
+function Rarity:grabAura()
+	local rotrBuff = 407063 -- Rocks on the Rocks buff
+	local unit = "player"
+	Rarity:Debug("Checking aura on the player character.")
+	local hasOpenedCOMGToday = IsQuestFlaggedCompleted(7325)
+	if not hasOpenedCOMGToday then
```

Comment:
I would definitely move the lockout check to the front (early exit), similar to how it's done for the Disgusting Vault. That would at least reduce the indentation/nesting level by one and allows mentally "checking off" this case before reading the remaining logic.

Reply:
super random but somehow i deleted a 5 from the quest id???? no idea how i managed that, it supposed to be 75325